apps/src/code-studio/progressRedux.js

```diff
@@ -227,7 +227,10 @@ export const setIsSummaryView = isSummaryView => ({ type: SET_IS_SUMMARY_VIEW, i
 // Do we have one or more lockable stages
 export const hasLockableStages = state => state.stages.some(stage => stage.lockable);
 
+// TODO - should i combine all the different things that map stages?
 export const lessonNames = state => state.stages.map(stage => stage.name);
+export const categories = state => state.stages.map(stage => stage.flex_category);
+export const hasGroups = state => _.uniq(categories(state)).length > 1;
```

Comment:
I now have three different sets of things I map stages to. Most places just care about two of these, but I'm starting to wonder if I should just combine these all.